templates/forms/feedback.php

```diff
@@ -0,0 +1,57 @@
+<form method="POST">
+    <input type="hidden" name="csrf_token" value="<?php e($GLOBALS['csrf_token']) ?>" />
+
+    <p><?php e($label['Feedback_intro']) ?></p>
+
+        <p>
+            <label for="feedback_frage1"><?php e($label['Feedback_frage1']) ?></label></br>
```

Comment:
No element with id `feedback_fragen1` exists. Could we use

```html
<label>
  <input type="checkbox" name="frage1" <?php e($_POST['frage1']) == 'ja' ? 'checked' : '' ?>>
  <?php e($label['Feedback_frage1']) ?>
</label>
```

instead?

Reply:
yes, that is better.

Reply:
There should be a yes/no choice. I will keep the radio-buttons.